src/Umbraco.Web.UI.Client/src/packages/core/menu/variant-menu-structure-workspace-context-base.ts

```diff
@@ -1,20 +1,21 @@
 import type { UmbVariantStructureItemModel } from './types.js';
+import { UMB_VARIANT_MENU_STRUCTURE_WORKSPACE_CONTEXT } from './variant-menu-structure-workspace-context.context-token.js';
 import type { UmbTreeRepository, UmbTreeRootModel } from '@umbraco-cms/backoffice/tree';
 import { createExtensionApiByAlias } from '@umbraco-cms/backoffice/extension-registry';
 import { UmbContextBase } from '@umbraco-cms/backoffice/class-api';
-import { UMB_VARIANT_WORKSPACE_CONTEXT } from '@umbraco-cms/backoffice/workspace';
+import { UMB_VARIANT_TREE_ENTITY_WORKSPACE_CONTEXT } from '@umbraco-cms/backoffice/workspace';
 import { UmbArrayState, UmbObjectState } from '@umbraco-cms/backoffice/observable-api';
 import type { UmbControllerHost } from '@umbraco-cms/backoffice/controller-api';
 import { UmbAncestorsEntityContext } from '@umbraco-cms/backoffice/entity';
 
-interface UmbMenuVariantTreeStructureWorkspaceContextBaseArgs {
+interface UmbVariantMenuStructureWorkspaceContextBaseArgs {
 	treeRepositoryAlias: string;
 }
 
-export abstract class UmbMenuVariantTreeStructureWorkspaceContextBase extends UmbContextBase {
```

Comment:
I believe we should stick to the old names for the classes, including "Tree." At some point, we will also have an implementation for a "collection/flat" structure.